src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -202,8 +202,7 @@ end
 -- Returns the simple name of a card given its ID.  This will find the card and strip any trailing
 -- SCED-specific suffixes such as (Taboo) or (Level)
 function getCardName(cardId)
-  local configuration = getConfiguration()
-  local allCardsBag = getObjectFromGUID(configuration.card_bag_guid)
+  local allCardsBag = getObjectFromGUID("15bb07")
```

Comment:
It actually does need to be here :)  Since the refactoring, DeckImporterMain doesn't have a reference to the configuration token any more.  It only matters for this one use (getting names for Raven Quill) but this was broken.

I didn't make it a constant since it's a single, well-known use that's obvious from the context, and one of my to-dos is to extract an API for AllCardsBag which would replace it.